### DISCUSSION TITLE
v3.0.0-avalonia11-preview3

### DISCUSSION THREAD

**Discussion by SKProCH** [AvaloniaCommunity/Material.Avalonia#210] [Announcements]
- **Fixes (adds a workaround) runtime theme changing bug**
- Add IReadOnlyTheme and change CurrentTheme to use it
- Move some styles to ControlTheme
- Improve some styles to proper work with Avalonia 11
- Make compability styles included by default
- Removed default font size for TextBlock to allow customizing it as part of other controls #208 #186
- Rework ProgressBar styles, add ShowProgressText support
- Fix scroll helper buttons content. #206
- Fixes TabControl style to support TabControlAssist.TabHeaderAlignment #203
- Rework first MenuItems, fix command execution for empty MenuItems #201, reformat MenuItem.xaml
- Fix Ripple effect remains after being simultaneously triggered

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaCommunity/Material.Avalonia/releases/tag/3.0.0-avalonia11-preview3'>v3.0.0-avalonia11-preview3</a>.</em>

**Comment by flarive** [comment]
Hey great news, it's so cool to be able to use Material Avalonia with Avalonia UI 0.11 !

I'm the maintainer of Neumorphism Avalonia (i started this project as a fork of your great Material Avalonia).
https://github.com/flarive/Neumorphism.Avalonia

I wanted to ask you if it was a big deal to make Material Avalonia working with Avalonia 0.11 previews ?

I will have to work on Neumorphism Avalonia soon to make it work with 0.11 and i would like to have an idea of the amount of work :)

Thanks a lot and great job !

**Reply by SKProCH** [reply]
Hello. Initially this is not a hard work, I've got a few compilation errors due to new APIs. But moving Styles to Themes will require some time.
